### PR TITLE
Increase mem threshold for arista 7800 devices

### LINKED ISSUE #16104: [Bug][202405]: System cpu and memory usage check fails
### Issue Description

The current memory usage threshold is 60% and that is exceeded in our Arista 7800 devices when configured in a T2 topology. Hence we need to increase this threshold to let the tests pass. 

This can be reproduced by running: `platform_tests/test_cpu_memory_usage.py::test_cpu_memory_usage::test_cpu_memory_usage`

### Results you see

```
Failed: system cpu and memory usage check fails
--------------------------------------------------
	 platform_tests 1
--------------------------------------------------
	 2024-12-11 14:58:19: platform_tests/test_cpu_memory_usage.py::test_cpu_memory_usage::test_cpu_memory_usage[cmp227-4]
```

### Results you expected to see

We expect the test to pass

### Is it platform specific

generic

### Relevant log output

_No response_

### Output of `show version`

_No response_

### Attach files (if any)

_No response_

### PR DESCRIPTION
**Pull request by arista-hpandya**
### Description of PR

Summary:
Fixes #16104

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
We wish to increase the mem threshold for Arista 7800 devices to 90 in order to help the platform_tests pass.

#### How did you do it?
Modified the threshold in `tests/platform_tests/test_cpu_memory_usage.py`

#### How did you verify/test it?
Pending testing

### CODE REVIEW

File: tests/platform_tests/test_cpu_memory_usage.py [hunk 1]
```diff
@@ -36,7 +36,8 @@ def setup_thresholds(duthosts, enum_rand_one_per_hwsku_hostname):
     high_cpu_consume_procs = {}
     is_asan = is_asan_image(duthosts, enum_rand_one_per_hwsku_hostname)
     if duthost.facts['platform'] in ('x86_64-arista_7050_qx32', 'x86_64-kvm_x86_64-r0', 'x86_64-arista_7050_qx32s',
-                                     'x86_64-cel_e1031-r0', 'x86_64-arista_7800r3a_36dm2_lc') or is_asan:
+       'x86_64-cel_e1031-r0') or is_asan or \
+       ('arista_7800' in duthost.facts['platform'].lower()):
```
Comment (arlakshm) [inline tests/platform_tests/test_cpu_memory_usage.py:40]: why do we need to increase to 90% if the steady state memory util is 60%? Can we increase it to 75%?

**Comment by arista-hpandya** [comment]
The pipeline failure looks unrelated to the change, re-triggering the build:

```
Error type: PREPARE_TESTBED_FAILED
2024-12-18T02:16:04.6091837Z Error message: Prepare testbed failed: testbed_q_sonic-elastictest-prod-vmss-D8s-v3_356439 for PREPARE_TESTBED_FAILED, testbed_id is: 67622fafde92dde8b33d6e46, cmd is: Checkout sonic-mgmt repo
2024-12-18T02:16:04.6093149Z Operation failed with exception: Exception('Test plan id: 67622e03b4abf968292cbf9e, status: FAILED, result: FAILED, Elapsed 120 seconds. Check https://elastictest.org/scheduler/testplan/67622e03b4abf968292cbf9e for test plan status')
```

File: tests/platform_tests/test_cpu_memory_usage.py (hunk 1 above)
Comment (arista-hpandya) [reply tests/platform_tests/test_cpu_memory_usage.py]: The initial thought was to set it at 90 as a preventative measure to ensure we don't hit the limit. But we can set it at 75%

**Review by arlakshm — APPROVED** [review]

**Pull request merged**